backend.py

```diff
@@ -1439,55 +1439,13 @@ def configureNetworking(mounts, admin_iface, admin_bridge, admin_config, hn_conf
     # Clean install only below this point
 
 
-    network_scripts_dir = os.path.join(mounts['root'], 'etc/sysconfig/network-scripts')
-
-    # remove any files that may be present in the filesystem already,
-    # particularly those created by kudzu:
-    network_scripts = os.listdir(network_scripts_dir)
-    for s in network_scripts:
-        if s.startswith('ifcfg-'):
-            os.unlink(os.path.join(network_scripts_dir, s))
-
-    # write the configuration file for the loopback interface
-    lo = open(os.path.join(network_scripts_dir, 'ifcfg-lo'), 'w')
-    lo.write("DEVICE=lo\n")
-    lo.write("IPADDR=127.0.0.1\n")
-    lo.write("NETMASK=255.0.0.0\n")
-    lo.write("NETWORK=127.0.0.0\n")
-    lo.write("BROADCAST=127.255.255.255\n")
-    lo.write("ONBOOT=yes\n")
-    lo.write("NAME=loopback\n")
-    lo.close()
-
-    save_dir = os.path.join(mounts['root'], constants.FIRSTBOOT_DATA_DIR, 'initial-ifcfg')
-    util.assertDir(save_dir)
-
-    # now we need to write /etc/sysconfig/network
-    nfd = open("%s/etc/sysconfig/network" % mounts["root"], "w")
-    nfd.write("NETWORKING=yes\n")
     if admin_config.modev6:
-        nfd.write("NETWORKING_IPV6=yes\n")
         util.runCmd2(['chroot', mounts['root'], 'systemctl', 'enable', 'ip6tables'])
     else:
-        nfd.write("NETWORKING_IPV6=no\n")
         netutil.disable_ipv6_module(mounts["root"])
-    nfd.write("IPV6_AUTOCONF=no\n")
-    nfd.write('NTPSERVERARGS="iburst prefer"\n')
```

Comment:
Should these args now be set in /etc/chrony.conf?

Reply:
we do not set `server` option in `/etc/chrony.conf`, 

It is set in `dhclient-script`, and I will raise a PR there.